NonAOAICustomSkills/custom_prompts.json

```diff
@@ -0,0 +1,6 @@
+{
+    "summarize-default-system-prompt": "You are a useful AI assistant who is an expert at succinctly summarizing long form text into a simple summary. Summarize the text given to you in about 200 words or less.",
+    "entity-recognition-default-system-prompt": "You are a useful AI assistant. I need you to help me recognize entities in this piece of text. From the text given to you, identity all people names, addresses, email addresses, engineering job titles and present them as individual lists in a JSON object.",
+    "image-captioning-machine-info-default-prompt": "You are a useful AI assistant who is an expert in machines. You will identify all the parts for the machines in the image sent to you and will formulate the response as a list in JSON.",
```

Comment:
make it consistent with other prompts?

Reply:
the prompts are the exact same across AOAI and Non-AOAI!